patterns/acu_data_compressed.hexpat

```diff
@@ -0,0 +1,49 @@
+#pragma description Assassin's Creed: Unity's Compressed .data file

```

Comment:
Since all of these files are are related to Assassin's Creed, I think it would make sense to put them all in a subfolder (like the Final Fantasy ones). Maybe under `patterns/ac_unity` or something similar